docs/user-guide/a1-06-spirv-target-specifics.md

```diff
@@ -0,0 +1,333 @@
+---
+layout: user-guide
+permalink: /user-guide/target-specifics
+---
+
+SPIR-V specific functionalities
+===============================
+
+This chapter provides information for SPIR-V specific functionalities and behaviors.
+
+Experimental support for the older versions of SPIR-V
+-----------------------------------------------------
+
+Support for SPIR-V 1.0, 1.1 and 1.2 is still experimental. When targeting the older SPIR-V profiles, Slang may produce SPIR-V that uses the instructions and keywords that were introduced in the later versions of SPIR-V.
```

Comment:
We should say that Slang's SPIRV backend is stable when emitting SPIRV 1.3 and later, however, support for SPIR-V 1.0, 1.1 and 1.2 is still experimental.